gossip/hub/instance_test.go

```diff
@@ -56,7 +56,7 @@ func TestConfigFromSingleFile(t *testing.T) {
 	tests := []struct {
 		desc, contents string
 		missingFile    bool
-		want           configpb.HubMultiConfig
+		want           *configpb.HubMultiConfig
```

Comment:
Why does this need to change?  I can't immediately see anywhere where `test.want` gets passed by value

Reply:
The loop over the tests does it via the assignment to the loop variable. That copies the outer struct, which copies the embedded proto.

Reply:
Ah, got it, thanks.